pset-functions.js

```diff
@@ -91,6 +116,30 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function wordCount(phrase){
+  let words = phrase.split(/[ ,]+/)
+  let wordObj = {}
+
+  words.forEach(function(element){
```

Comment:
@kertcodes everything looks fine, the minor issue here is the syntax of the `if..else` statements, best practice is to structure them as follows:

```
```
if (condition) {

} else if (condition) { //<-- as many as needed

} else {

}
```